doc/source/intro.rst

```diff
@@ -18,67 +18,77 @@ because it keeps Python packages isolated from your system Python.
 Usage
 ^^^^^
 
-Start ACP
-~~~~~~~~~
+Tutorial setup
+~~~~~~~~~~~~~~
 
-Start a Python interpreter and import the PyACP package:
+Start a python interpreter and import the required PyACP packages:
+
+.. testcode::
+
+    import ansys.acp.core as pyacp
+    from ansys.acp.core.extras import (
+        ExampleKeys,
+        get_example_file,
+    )  # This is only required for the tutorial
+
+To not pollute the filesystem, we are going to create a temporary directory were we can download the files:
 
 .. testcode::
-    :hide:
 
     import tempfile
-    import shutil
     import pathlib
-    import os
 
-    workdir_doctest = tempfile.TemporaryDirectory()
-    DATA_DIRECTORY = pathlib.Path(workdir_doctest.name)
-    _ = shutil.copyfile(
-        "../tests/data/minimal_complete_model_no_matml_link.acph5",
-        DATA_DIRECTORY / "model.acph5",
-    )
-    _ = shutil.copyfile("../tests/data/minimal_model_2.cdb", DATA_DIRECTORY / "model.cdb")
-    old_cwd = os.getcwd()
-    os.chdir(DATA_DIRECTORY)
+    tempdir = tempfile.TemporaryDirectory()
+    WORKING_DIR = pathlib.Path(tempdir.name)
+
+Download the example files by using the provided helper function as:
 
 .. testcode::
 
-    import ansys.acp.core as pyacp
+    plate_no_matml_path = get_example_file(
+        ExampleKeys.GS_MINIMAL_PLATE_NO_MATML, WORKING_DIR
+    )
+    plate_path = get_example_file(ExampleKeys.GS_MINIMAL_PLATE, WORKING_DIR)
+
+Start ACP
+~~~~~~~~~
 
 Next, start an ACP instance:
 
 .. testcode::
 
     acp = pyacp.launch_acp()
 
-Get a model
-~~~~~~~~~~~
+Load a model
+~~~~~~~~~~~~
 
 You can resume a model from an existing ACP DB (ACPH5) or built it from
-scratch by importing an FE model (mesh).
+scratch by importing an FE model (mesh). We are going to use the path to the model returned by the ``get_example_file``
+function, but a raw path like ``r"path\to\your\model.acph5"`` can also be used.
 
-To load an existing model with PyACP, use the :meth:`.import_model` method:
+To load an existing ACP layup model with PyACP, use the :meth:`.import_model` method:
 
 .. testcode::
 
-    model = acp.import_model("model.acph5")
+    plate_no_matml_model = acp.import_model(plate_no_matml_path)
```

Comment:
Here, we could for example use `model_from_acph5` and `model_from_cdb` as variable names.